cgroup_top.py

```diff
@@ -620,6 +666,8 @@ def on_keyboard(c):
             run(-2, ['docker', 'stop', selected_name])
         elif selected['type'] in ['lxc', 'lxc-user'] and HAS_LXC:
             run(selected['owner'], ['lxc-stop', '--name', selected_name, '--nokill', '--nowait'])
+        elif selected['type'] == 'openvz' and HAS_OPENVZ:
+            run(selected['owner'], ['vzctl', 'suspend', selected_name])
```

Comment:
I think this should rather be 'stop'. This said, If suspending from ctop is highly relevant for openvz containers (you are the one whos knows), feel free to add an additional action key + document it in README.